test/InvokeDynamic.java

```diff
@@ -24,8 +42,19 @@ public static void main(String[] args) {
   }
 
   private void test() {
-    int c = 2;
-    Operation op = (a, b) -> ((a + b) * c) - foo;
-    expect(op.operate(2, 3) == ((2 + 3) * 2) - foo);
+    { int c = 2;
+      Operation op = (a, b) -> ((a + b) * c) - foo;
```

Comment:
Will this test get AOT-compiled at all?  I thought it was just the classpath.  If not, this may not yield good coverage.